cluster/images/pkg-unpack/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM BASEIMAGE
```

Comment:
Small update here, I did a little spelunking and it appears we are using [alpine:3.7](https://github.com/upbound/build/blob/af023ee602ef60d14612f06f91c6fcb38c402269/makelib/image.mk#L33) as base image and using builder pattern as opposed to multistage builds (i.e. build binary in [`cross` image](https://github.com/upbound/build/tree/master/cross). This isn't terrible, but I think we should use distroless instead of alpine for most of our base images (you can override base image when using `build` submodule). @kasey pointed out for this image we can likely just use scratch because we don't need think like ca certs.

Reply:
I actually attempted doing that when I was integrating with build submodule. However, I realized there is [no official](https://github.com/GoogleContainerTools/distroless/issues/377) `arm64` builds of distroless images. So, I just gave up and kept using the existing default.

Reply:
It looks like it [may be possible](https://github.com/GoogleContainerTools/distroless/issues/346) to use distroless on multiple platforms, but I am fine with deferring this to a future enhancement 👍